src/rust/ide/view/src/documentation.rs

```diff
@@ -108,7 +119,11 @@ impl ViewModel {
         display_object.add_child(&dom);
         display_object.add_child(&overlay);
         scene.dom.layers.front.manage(&dom);
-        ViewModel {logger,dom,size,overlay,display_object}.init()
+
+        let closures: Vec<Closure<_>> = Vec::new();
+        let closures = CloneCell::new(closures);
+        let closures = Rc::new(closures);
```

Comment:
not aligned `=` . Change the first line to `let closures = Vec::<Closure<_>>::new();`.